python/tools/dht/tests.py

```diff
@@ -371,6 +478,9 @@ def _gottaGetThemAllPokeNodes(self, consumer, hashes, nodes=None):
                 for n in DhtFeatureTest.foreignNodes:
                     nodes.add(n)
 
+    def __normalConditionTest(self):
```

Comment:
Also, I see that you have copied the code from `_totallyNormalTest` into another function (`_normalConditionTest`) as if you wanted to rename it, but you have not deleted the [old one](https://github.com/savoirfairelinux/opendht/pull/56/files#diff-7590f949505d55358a56a80e319fe3baR597) (line 597).